### DISCUSSION TITLE
ورود لیست آیپی از فایل تکست در ورژن ویندوزی

### DISCUSSION THREAD

**Discussion by yangol32** [MortezaBashsiz/CFScanner#549] [Ideas]
تا قبل از آپدیت اخیر من داخل یکی از فایلهای تکست نتیجه ، لیست آیپی دلخواهم (نه رنج آیپی) را میگذاشتم و ایمپورت میکردم. ولی از آپدیت اخیر با اضافه شدن عدد آپلود به فایل نتیجه دیگه وارد نمیکنه. اگه قابلیتی بگذارید که لیست آیپی دلخواه از فایل تکست معمولی وارد کنه خوب میشه.

**Comment by boyluti** [comment]
لطفا تو کانال تلگرام مرتضی  نسخه ویندوز رو آپدیت کنین.هر چ زودتر. با تشکرر فراوانن.

**Comment by goingfine** [comment]
اخرین نسخه رو بگیرید این مساله حل شده
